velox/expression/ExprUtils.h

```diff
@@ -15,10 +15,27 @@
  */
 #pragma once
 
-#include "velox/expression/Expr.h"
+#include "velox/core/Expressions.h"
 
 namespace facebook::velox::expression::utils {
 
+/// Indicates the type of inputs of an expression.
+enum class ExprInputsKind {
+  /// kAllConstant: All inputs are of constant kind.
+  kAllConstant = 0,
+  /// kAllField: All inputs are of field kind.
```

Comment:
"of constant kind" and "of field kind" reads as broken English. Please, consider editing to 

- All inputs are constant.
- All inputs are field references.
- Each input is either constant or field reference.
- ...